samples/openapi3/client/petstore/go/go-petstore/README.md

```diff
@@ -123,6 +123,8 @@ Class | Method | HTTP request | Description
 ## Documentation For Models
 
  - [AdditionalPropertiesClass](docs/AdditionalPropertiesClass.md)
+ - [AllOfPrimitiveTypes](docs/AllOfPrimitiveTypes.md)
```

Comment:
I think we see here other more important issue:
- We have 2 files with exactly the same content
Original Model and ModelAllOf -  are exactly the same. 

@wing328 Do you know why we need that separate object? It would not be used in generated code and it contains broken import.

Maybe we can use transformer abstraction to hide such cases?